builder/parser/parser.go

```diff
@@ -61,6 +61,7 @@ func init() {
 		command.Entrypoint: parseMaybeJSON,
 		command.Expose:     parseStringsWhitespaceDelimited,
 		command.Volume:     parseMaybeJSONToList,
+		command.Arg:        parseNameOrNameVal,
```

Comment:
I don't see why it can't be verified in the parser, but honestly it doesn't matter that much